packages/contracts-periphery/package.json

```
@@ -60,20 +60,18 @@
     "@nomiclabs/hardhat-ethers": "^2.0.2",
     "@nomiclabs/hardhat-etherscan": "^3.0.3",
     "@nomiclabs/hardhat-waffle": "^2.0.3",
-    "@rari-capital/solmate": "7.0.0-alpha.3",
     "@openzeppelin/contracts": "4.7.3",
     "@openzeppelin/contracts-upgradeable": "4.7.3",
-    "@types/node": "^17.0.21",
+    "@rari-capital/solmate": "7.0.0-alpha.3",
```

Can we also change this to `transmissions11`
```suggestion
    "@transmissions11/solmate": "7.0.0-alpha.3",
```

---

hmm on latest it's just called `solmate` https://github.com/transmissions11/solmate/blob/main/package.json